wordpress-mobile/WordPress-FluxC-Android
# Coupons - Paging support

This PR adds paging support to for fetching coupons. 

**To test:**
1. Make sure your test store has > 3 coupons
2. Start the example app and go to Woo -> Coupons
3. Tap on the Fetch coupons button
4. Notice the 1st page of 3 coupons is fetched
5. Keep tapping the button, the list of fetched coupons will grow
6. When all coupons are fetched, notice the button gets disabled
